### PR TITLE
[event] hotfix of event migration

### PR DESCRIPTION
**Pull request by lightmark**
## Type of Change
- [ ] New feature
- [x] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [x] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by davidiw — APPROVED** [review]

File: aptos-move/framework/aptos-framework/sources/delegation_pool.move [hunk 1]
```diff
@@ -1412,6 +1506,17 @@ module aptos_framework::delegation_pool {
...
+                AddStakeEvent {
+                    pool_address,
+                    delegator_address,
+                    amount_added: amount,
+                    add_stake_fee,
+                },
+            );
+        };
+
         event::emit_event(
```
Comment (runtian-zhou) [inline aptos-move/framework/aptos-framework/sources/delegation_pool.move:1520]: Do we need to emit both events after migration?

Comment (lightmark) [reply aptos-move/framework/aptos-framework/sources/delegation_pool.move:1520]: We delete v1 afterwards.

Comment (runtian-zhou) [reply aptos-move/framework/aptos-framework/sources/delegation_pool.move:1520]: Afterwards meaning next release? Does it make sense to make old logic dependent on the flag not set so we won't have a period where we fire two events at the same time?

Comment (lightmark) [reply aptos-move/framework/aptos-framework/sources/delegation_pool.move:1520]: we must have a period that both exist. like rolling update....

**Review by runtian-zhou — APPROVED** [review]

**Pull request merged**